data/projects/infrastructure/uyuni-project.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: Uyuni Project
```

Comment:
```suggestion
name: Uyuni
```

And same for the filiname, FMPOV.

The name is in fact "Uyuni". Project was only added because there was a GitHub user, a twitter user, and a domain.

But other than that, the [site](https://www.uyuni-project.org/), the doc, and all communications are always mentioning

Reply:
Change but my problem with Uyuni is it's a horrible name SEO-wise. We can never Google-win over the actual salt lake in Bolivia :-(

Reply:
BTW: Is `infrastructure` the right category? Or maybe `operations`? So it covers more than one category, as suggested by the readme at this repository?

Reply:
SUSE Manager is sold as infrastructure but I had exactly the same doubt. And since Trento and Rancher are under operations, moving there might be a good idea.

Reply:
It'd be cool if we had labels (multiple labels allowed) instead of just one category.

Reply:
that's a great idea, and something i would like to have for the future. please open an issue and suggest it, and we'll continue the discussion there.